lib/internal/errors.js

```diff
@@ -136,6 +139,14 @@ const maybeOverridePrepareStackTrace = (globalThis, error, trace) => {
   return kNoOverride;
 };
 
+const chainErrors = (error, originalError) => {
+  if (error) {
+    error.cause = originalError;
+    return error;
+  }
+  return originalError;
+};
```

Comment:
Should this be: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/AggregateError